rdoc.gemspec

```diff
@@ -28,6 +28,10 @@ RDoc includes the +rdoc+ and +ri+ tools for generating and displaying documentat
   s.homepage = "https://ruby.github.io/rdoc"
   s.licenses = ["Ruby"]
 
+  s.metadata["homepage_uri"] = s.homepage
```

Comment:
Doesn’t rubygems.org fallback to `#homepage`?

Reply:
Yeah, I think a lot of tools favour `homepage` over `metadata`, but I thought it wouldn't harm to add it. Do you prefer not adding `homepage_uri`?

Reply:
Just I was curious a little.

Reply:
`metadata.homepage_uri` value is going to be copied from `homepage` when `metadata.homepage_uri` was not given. So, it'd be better to keep `metadata.homepage_uri` empty if there's no special reason to specify both. We don't need to repeat what RubyGems internally does.

Reply:
see: https://github.com/rubygems/rubygems/blob/57b6c04dc2aac0607d52468918b45babadd0e10d/lib/rubygems/specification_policy.rb#L411-L421

Reply:
@amatsuda That method looks like just validating `Specification#homepage`, but nothing about `metadata["homepage_uri"]`, no?

Reply:
Indeed! My bad, please forget about my comment above...!